sechub-doc/src/test/java/com/mercedesbenz/sechub/restdoc/SchedulerRestControllerRestDocTest.java

```diff
@@ -491,6 +496,59 @@ public UUID getUUID() {
         /* @formatter:on */
     }
 
+    @Test
+    @UseCaseRestDoc(useCase = UseCaseUserUploadsBinaries.class)
+    public void restDoc_userUploadsBinaries() throws Exception {
+        /* prepare */
+        String apiEndpoint = https(PORT_USED).buildUploadBinariesUrl(PROJECT_ID.pathElement(), JOB_UUID.pathElement());
+        Class<? extends Annotation> useCase = UseCaseUserUploadsBinaries.class;
+
+        ScheduleSecHubJob job = new ScheduleSecHubJob() {
+            public UUID getUUID() {
+                return randomUUID;
+            };
+        };
+        job.setExecutionResult(ExecutionResult.OK);
+        job.setStarted(LocalDateTime.now().minusMinutes(15));
+        job.setEnded(LocalDateTime.now());
+        job.setExecutionState(ExecutionState.INITIALIZING);
+        job.setOwner("CREATOR1");
+        job.setTrafficLight(TrafficLight.GREEN);
+
+        ScheduleJobStatus status = new ScheduleJobStatus(job);
+
+        when(mockedScheduleJobStatusService.getJobStatus(PROJECT1_ID, randomUUID)).thenReturn(status);
+
+        InputStream inputStreamTo = RestDocTestFileSupport.getTestfileSupport().getInputStreamTo("upload/tarfile_contains_only_test1.txt.tar");
+        MockMultipartFile file1 = new MockMultipartFile("file", inputStreamTo);
+        /* execute + test @formatter:off */
+        this.mockMvc.perform(
+                multipart(apiEndpoint, PROJECT1_ID,randomUUID).
+                    file(file1).param("checkSum", "mychecksum")
+                ).
+                    andExpect(status().isOk()).
+                            // https://docs.spring.io/spring-restdocs/docs/2.0.2.RELEASE/reference/html5/
+                    andDo(defineRestService().
+                            with().
+                                useCaseData(useCase).
+                                tag(RestDocFactory.extractTag(apiEndpoint)).
+                            and().
+                            document(
+                                    pathParameters(
+                                            parameterWithName("projectId").description("The id of the project where sourcecode shall be uploaded for"),
+                                            parameterWithName("jobUUID").description("The jobUUID for sechub job")
```

Comment:
I changed the suggested text to 
> The SecHub jobUUID. During the job creation this unique job identifier is automatically generated by SecHub.

The origin phrase is used multiple times. So I introduced a constant to avoid repeates